<launch>

  <!--rviz中的fixed frame-->
  <arg name="fixed_frame" default="velodyne" />
  <!--radar x方向偏移量-->
  <arg name="x_offset" default="1.6" />

  <!--连接VLP16-->
  <include file="$(find velodyne_pointcloud)/launch/VLP16_points.launch" />
  
  <!--接收MPC单片机CAN消息-->
  <node pkg="socketcan_bridge" type="socketcan_to_topic_node" name="socketcan_to_topic_node" />
  <node pkg="raw_data" type="MPC_raw" name="MPC_raw" >
    <param name="fixed_frame" value="$(arg fixed_frame)" />
    <param name="x_offset" value="$(arg x_offset)" />
  </node>

  <!--rviz可视化-->
  <node pkg="rviz" type="rviz" name="rviz" />   

  <!--rqt_plot显示曲线-->
  <node pkg="rqt_plot" type="rqt_plot" name="plot1" />

</launch>